roles/common/tasks/main.yml

```diff
@@ -34,7 +34,7 @@
     line: '{{ hostvars[item].int_if.ipaddr }} {{item}}' 
     state: present
   #when: hostvars[item].int_if.ipaddr is defined
```

Comment:
Are you intending to leave this line commented out?  Since you are running against the 'openstack' group instead of 'all', we should no longer need the 'when' since all openstack hosts should have that var defined.  If we don't need the line, just remove it.

Not 'all' hosts have int_if defined so the task was failing for me during deployment with this line commented out.